core/components/organisms/verticalNav/MenuItem.tsx

```diff
@@ -25,29 +27,44 @@ interface MenuLabelProps {
   label: string;
   disabled?: boolean;
   isActive: boolean;
+  expanded?: boolean;
 }
 
 interface MenuIconProps {
   isChildrenVisible?: boolean;
   isActive?: boolean;
   disabled?: boolean;
+  showClosingAnimation?: boolean;
 }
 
 const MenuLabel = (props: MenuLabelProps) => {
-  const { label, disabled, isActive } = props;
+  const { label, expanded, disabled, isActive } = props;
+
+  const LabelClass = classNames({
+    ['MenuItem-animate']: true,
+    ['MenuItem-expanded-label']: expanded,
+    ['MenuItem-collapsed-label']: !expanded,
```

Comment:
Please change the className `MenuItem-collapsed-label` to `MenuItem-label--collapsed` &  `MenuItem-label--expanded`

Reply:
ok will do